tortoise/metrics/prometheus.go

```diff
@@ -0,0 +1,23 @@
+package metrics
+
+import (
+	"github.com/spacemeshos/go-spacemesh/metrics"
+)
+
+const (
+	// Subsystem is a subsystem shared by all metrics exposed by this package.
+	Subsystem = "tortoise"
```

Comment:
i am not really sure i appreciate this Subsystem thing... if i am looking for a metrics, do i need to query by subsystem? i will have no clue/memory of which subsystem a metrics belong to.

Reply:
What would you suggest to use instead?

Reply:
do you know the purpose of subsystem? is it used for querying the metrics? 
anyhow, not important. we can fix things if there is UX issues

Reply:
As I understand, it's needed for optional grouping and may be left blank